### DISCUSSION TITLE
Internal Server error version 0.51-alpha1

### DISCUSSION THREAD

**Discussion by PH5HP** [rejetto/hfs#411] [General]
Hi,

I got an error starting up the server after installing 0.51-alpha1. See my screenshot PSE.

Kind regards,
Henk - fileserver.ph5hp.nl

![image](https://github.com/rejetto/hfs/assets/140949730/26c514e7-c096-474f-8c74-286572545b89)

**Comment by rejetto** [comment]
is it working despite that?

**Reply by PH5HP** [reply]
Hi,

Second time I started now ... It has no errors anymore... No idea why.

Kind regards,
Henk - fileserver.ph5hp.nl

**Reply by PH5HP** [reply]
Hi,

I spoke to soon ... Lots of errors now and in the browser I got only the line at the top: 'Internal server error'

Kind regards,
Henk - fileserver.ph5hp.nl

**Comment by rejetto** [comment]
i'm working on it, i will deliver probably today

**Comment by rejetto** [comment]
ok, alpha2 is out and should fix that and several other problems

**Reply by PH5HP** [reply]
Hi,

No errors (yet) :-)

Kind regards,
Henk - fileserver.ph5hp.nl

**Reply by PH5HP** [reply]
Hi,

New error after a while. See PSE my screenshot. It seems de server is still running.

Kind regards,
Henk - fileserver.ph5hp.nl
![image](https://github.com/rejetto/hfs/assets/140949730/0ccdf9b7-eeb1-4285-9aba-e8e31a3f83f9)

**Comment by rejetto** [comment]
thanks for your feedback!
i decided to retire alpha2, replaced by alpha3

**Reply by PH5HP** [reply]
Hi,

Running for several hours now and still no errors on the screen. The only thing I noticed is that the plugin starfield-background does not work anymore, but that can also be the case with the last stable version. I think that one still worked with version 0.50.0.

Kind regards,
Henk - fileserver.ph5hp.nl - www.ph5hp.nl

**Comment by rejetto** [comment]
starfield working for me on 0.51.0-alpha3

**Reply by PH5HP** [reply]
Hi,

Not here. I do not know why. Not a big issue, but I like that plugin. I did not changed it. Now I tried to disable it and then enable it again. Still no luck in the browser here. I can see that if I refresh my server a brief moment it is visble and directly after that nothing anymore.

You are free to try my server. It is possible it is on my browserside that it is not displayed.

https://fileserver.ph5hp.nl

Kind regards,
Henk

**Reply by rejetto** [reply]
i guess you solved it :)
of course it has something to do with your customizations
<img width="420" alt="image" src="https://github.com/rejetto/hfs/assets/1367199/02fe254c-9765-4691-95f5-39ca16e5296b">

**Reply by PH5HP** [reply]
Hi,

It is a browser setting I suppose. It still does not work in Chrome anymore, but it works in in Edge. Hmm... Something to sort out. I did not change anything in Chrome. Very strange. Sorry for the inconvience.

Kind regards,
Henk - fileserver.ph5hp.nl

**Comment by rejetto** [comment]
i don't think it's a setting, more likely an extension.
try in incognito

**Reply by PH5HP** [reply]
Hi,

I solved it. I renamed my server some days ago and my browser extension which sets all in darkmode did not have this URL yet. Adding the exeption again solved it ... Sorry voor the inconvience...

Kind regards,
Henk - fileserver.ph5hp.nl
